uiowa/uiowa
# Paragraphs carousel

Carousel implementation for https://github.com/uiowa/uiowa/issues/1660 and https://github.com/uiowa/uids/issues/123

While I am happy with where this is, I don't think this is done. We probably want to test and check sizing and possibly what we want to do with the edge to edge display.


TO TEST:

load up any site with uids_base as the theme, and create a paragraphs carousel. you should be  able to see a styled carousel made with slick.
